WordPressAuthenticator/Signin/LoginPrologueLoginMethodViewController.swift

```diff
@@ -1,6 +1,11 @@
 import WordPressUI
 import WordPressShared
 
+
+/// This class houses the "3 button view":
+/// Continue with WordPress.com, Continue with Google, Continue with Apple
+/// and a text link - Or log in by entering your site address.
+///
 class LoginPrologueLoginMethodViewController: NUXViewController {
```

Comment:
I'll throw my 2 cents in here. This is an existing class in the existing auth flow. I think we should leave it as-is as it will be defunct "soon", and will just be wasted effort to rename and fix what uses it. Also, I'll probably have to refactor/reimplement it anyway for the [Get Started](https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/issues/404) view. So, either way, this will be resolved eventually.

Reply:
Oh ok. I was not aware of the upcoming work. In this case it makes sense to leave it as it is, thanks @ScoutHarris !